packages/inter-protocol/test/price/test-fluxAggregator.js

```diff
@@ -0,0 +1,705 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test as unknownTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+
+import { makeMockChainStorageRoot } from '@agoric/internal/src/storage-test-utils.js';
+import { subscribeEach } from '@agoric/notifier';
+import {
+  eventLoopIteration,
+  makeFakeMarshaller,
+} from '@agoric/notifier/tools/testSupports.js';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { setupZCFTest } from '@agoric/zoe/test/unitTests/zcf/setupZcfTest.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { provideFluxAggregator } from '../../src/price/fluxAggregator.js';
+import { topicPath } from '../supports.js';
+
+/** @type {import('ava').TestFn<Awaited<ReturnType<typeof makeContext>>>} */
+const test = unknownTest;
+
+const defaultConfig = {
+  maxSubmissionCount: 1000,
+  minSubmissionCount: 2,
+  restartDelay: 5,
+  timeout: 10,
+  minSubmissionValue: 100,
+  maxSubmissionValue: 10000,
+};
+
+const makeContext = async () => {
+  const link = makeIssuerKit('$LINK', AssetKind.NAT);
+  const usd = makeIssuerKit('$USD', AssetKind.NAT);
+
+  async function makeChainlinkAggregator(config) {
+    const terms = { ...config, brandIn: link.brand, brandOut: usd.brand };
+    const zcfTestKit = await setupZCFTest(undefined, terms);
+
+    // ??? why do we need the Far here and not in VaultFactory tests?
+    const marshaller = Far('fake marshaller', { ...makeFakeMarshaller() });
+    const mockStorageRoot = makeMockChainStorageRoot();
+    const storageNode = E(mockStorageRoot).makeChildNode('priceAggregator');
+
+    const manualTimer = buildManualTimer(() => {});
+
+    const baggage = makeScalarBigMapStore('test baggage');
+    const quoteIssuerKit = makeIssuerKit('quote', AssetKind.SET);
+
+    const aggregator = provideFluxAggregator(
+      baggage,
+      zcfTestKit.zcf,
+      manualTimer,
+      { ...quoteIssuerKit, assetKind: 'set', displayInfo: undefined },
+      await E(storageNode).makeChildNode('LINK-USD_price_feed'),
+      marshaller,
+    );
+
+    return { ...aggregator, manualTimer, mockStorageRoot };
+  }
+
+  return { makeChainlinkAggregator };
+};
+
+test.before('setup aggregator and oracles', async t => {
+  t.context = await makeContext();
+});
+
+test('basic', async t => {
+  const aggregator = await t.context.makeChainlinkAggregator(defaultConfig);
+  const oracleTimer = aggregator.manualTimer;
+
+  const pricePushAdminA = await E(aggregator.creatorFacet).initOracle(
+    'agorice1priceOracleA',
```

Comment:
I'm having trouble reading these strings.I parse them as "agoric", "e1", "priceOracle", "A". What's the "e1" about?  Why isn't the "E" capitalized?   It's just a doc-string, but my eyes stumble over it every time.

Reply:
`e1` is a typo. `agoric1` is how all addresses start. `priceOracleA` is meant to be the unique identifier. The reason to include `agoric1` is to convey that it is a (mock) BEC32 address.

Reply:
I was hoping it would be something like that.  thanks.